python_base/Dockerfile

```diff
@@ -69,7 +70,8 @@ RUN virtualenv /tmpvenv -p python${INSPIRE_PYTHON_VERSION} && \
     deactivate && \
     mv /tmpvenv/src /src-cache && \
     rm -rf /tmpvenv && \
-    mkdir /code
+    mkdir /code && \
+    dbus-uuidgen > /etc/machine-id
```

Comment:
If anyone else is wondering, apparently `firefox` needs a well-formed `machine-id` to start.